scipy/optimize/tests/test_cobyla.py

```diff
@@ -71,6 +80,48 @@ def c3(w):
         assert_(sol.maxcv > 1e-6)
         assert_(not sol.success)
 
+    def test_vector_constraints(self):
+        # test that fmin_cobyla and minimize can take a combination
+        # of constraints, some returning a number and others an array
+        def fun(x):
+            return (x[0] - 1)**2 + (x[1] - 2.5)**2
+
+        def fmin(x):
+            return fun(x) - 1
+
+        cons1 = Constraints([[1, -2, 2], [-1, -2, 6], [-1, 2, 2]])
+
+        def cons2(x):
+            return x     # identity, acts as bounds x > 0
+
+        x0 = np.array([2, 0])
+        cons_list = [fun, cons1, cons2]
+
+        class Solution:
+            pass
+        solution = Solution()
+        solution.x = [1.4, 1.7]
+        solution.fun = 0.8
```

Comment:
What's the point of these 5 lines? Just defined two variables named, e.g. `xsol` and `fsol`.